macos/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+# Tasks to be run to provision the macOS buildfarm
+
+# Homebrew stuff
+
+# We redirect stdin from /dev/null so it won't prompt for install.
+#- 
+#  name: "Installing Homebrew"
+#  shell: ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null
+# NOTE: Might be best to use https://github.com/geerlingguy/ansible-role-homebrew to avoid prompts etc.
+
```

Comment:
Yes lets test out that role and see if it gives us what we need. Ned to make sure its idempotent as well